### PR TITLE
[#155210839] Finalize links in the header

### PR DESCRIPTION
**Pull request by Yutna**
https://www.pivotaltracker.com/story/show/155210839

✅ Add links
✅ Add Medium link

desktop

mobile

### CODE REVIEW

File: src/assets/stylesheets/components/_app-header.scss [hunk 1]
```diff
@@ -80,10 +80,8 @@
     &-item-icon {
       @include transition(filter 0.3s ease);
 
-      width: 32px;
-      height: 32px;
-      margin-left: 1rem;
-      background-size:(70% 70%);
+      height: 22px;
```
Comment (olivierobert) [inline src/assets/stylesheets/components/_app-header.scss]: Why pixels here and not `rem`?

File: src/components/AppHeader/index.jsx [hunk 2]
```diff
@@ -11,21 +11,32 @@ const AppHeader = () => (
...
           </li>
           <li className="app-header__navbar-item-icon app-header__navbar-item-icon--twitter">
-            <a href="#twitter"><span className="sr-only">Follow Nimbl3 on Twitter</span></a>
+            <a href="https://twitter.com/gonimbl3" target="_blank" rel="noopener noreferrer">
+              <span className="sr-only">Follow Nimbl3 on Twitter</span>
+            </a>
           </li>
           <li className="app-header__navbar-item-icon app-header__navbar-item-icon--github">
-            <a href="#github"><span className="sr-only">Follow Nimbl3 on Github</span></a>
+            <a href="https://github.com/nimbl3" target="_blank" rel="noopener noreferrer">
```
Comment (olivierobert) [inline src/components/AppHeader/index.jsx]: How does `"@nimbl3` work for Github link?

File: src/components/AppHeader/index.jsx [hunk 3]
```diff
@@ -11,21 +11,32 @@ const AppHeader = () => (
...
+            <a href="https://nimbl3.com/" target="_blank" rel="noopener noreferrer">About us</a>
           </li>
           <li className="app-header__navbar-item-text">
-            <a href="#Join us">Join us</a>
+            <a href="http://workhiro.com/companies/nimbl3" target="_blank" rel="noopener noreferrer">Join us</a>
           </li>
         </ul>
         <ul className="app-header__navbar">
+          <li className="app-header__navbar-item-icon app-header__navbar-item-icon--medium">
+            <a href="https://www.facebook.com/nimbl3" target="_blank" rel="noopener noreferrer">
```
Comment (olivierobert) [inline src/components/AppHeader/index.jsx]: Good attention to details for using `rel="noopener"`

File: src/assets/stylesheets/components/_app-header.scss [hunk 4]
```diff
@@ -38,10 +38,10 @@
       border: none;
     }
   }
-  
+
   &__logo {
-    width: 110px;
-    height: 30px;
+    width: 6.88rem;
+    height: 1.88rem;
```
Comment (olivierobert) [inline src/assets/stylesheets/components/_app-header.scss]: How did you this conversion?

```
110px/16px = 6.875rem
30px/16px = 1.875rem
```

File: src/components/AppHeader/index.jsx (hunk 2 above)
Comment (Yutna) [reply src/components/AppHeader/index.jsx]: Change a link url to `https://github.com/@nimbl3` right ? I try it and It render 404 page.

**Pull request merged**